srcpkgs/gpodder/template

```diff
@@ -1,21 +1,22 @@
 # Template file for 'gpodder'
 pkgname=gpodder
-version=3.10.17
+version=3.10.21
 revision=1
 wrksrc="gpodder-${version}"
 hostmakedepends="python3 intltool"
 depends="eyeD3 gtk+3 hicolor-icon-theme python3-dbus python3-gobject
  python3-html5lib python3-mygpoclient python3-podcastparser"
-checkdepends="${depends} python3-MiniMock python3-coverage desktop-file-utils"
 short_desc="Podcast client"
 maintainer="bra1nwave <bra1nwave@protonmail.com>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/gpodder/gpodder"
 distfiles="${homepage}/archive/${version}.tar.gz"
-checksum=36a04e4d6a81f50b50d1f7691955d4f460e72f71b9519dad42b805a987434210
+checksum=014e619de64d3e3dc8493929af8007b3caf09dd77e153bf778f1708d55946878
 
 do_check() {
-	make releasetest
+	: # unittest require pytest-localserver which is not in the repos,
+	  # had its latest release in 2018 and has a defunct bitbucket repo
+	  # as its source repo.
```

Comment:
Could you link some sources to this? I can not find a single mention of pytest-localserver in the repository, do you mean pytest-httpserver? That one is quite active (last release 2 days ago) and I noticed it's quite common, so we could package it, but it will not be a requirement to merge this PR.

Reply:
pytest-localserver : https://pypi.org/project/pytest-localserver/
If pytest-httpserver provides the same fixture I can package it :)

Reply:
Done, thanks :)